### ISSUE TITLE
add meta for tickets  for slack

### ISSUE THREAD

**Issue by tjgillies** [ZenHubIO/support#1318]
When I paste a ticket url into slack i just get the useless 

> ZenHub unifies project management and development within GitHub.

without any meta. this seems easily solvable

**Comment by kendramoroz** [comment]
@tjgillies agreed! This is a great enhancement suggestion. Bumping over to the discussion pipeline. We'll keep the issue updated as more information comes available about enhancements to the way we handle Slack URLs.
